Engine/Objects/Actor.cpp

```diff
@@ -5279,7 +5279,10 @@ void SpawnEncounter(MapInfo *pMapInfo, SpawnPointMM7 *spawn, int a3, int a4, int
         Src = &pMonsterStats->pInfos[v28];
         strcpy(pMonster->pActorName, Src->pName);
         pMonster->sCurrentHP = Src->uHP;
-        assert(sizeof(MonsterInfo) == 88);
+
+        // assert(sizeof(MonsterInfo) == 88);
+        assert(sizeof(MonsterInfo) == 80 + 2 * sizeof(void *));
```

Comment:
is fread/memcpy of that addressed?

Reply:
MonsterInfo is addressed, but again I imagine you would prefer to use MonsterInfo_MM7? At that point what's the purpose of these asserts? Point them at _MM7 structure and ensure no changes are being made to these original structures?

Reply:
MonsterInfo_MM7 created